Jenkinsfile

```diff
@@ -123,12 +115,29 @@ def getVersion() {
     if (!version) {
         error "Version must be set"
     }
+    if (isMasterBranch() && !isSnapshotVersion(version)) {
```

Comment:
Should there be a check for snapshot preset in release branch, when we are checking for absence of snapshot in master?

Reply:
Good question. I was thinking it may make sense to have a release (e.g. after code-freeze) for testing but not release from it yet. Wdyt?

Reply:
Makes sense. Then maybe I would add a checkbox `Release`. If it's marked, it has to be a non-snapshot. Otherwise it must be a snapshot. I mean it's not a deal breaker I was just curious since your are checking it "one way", it might be worth checking the "other way" as well.

Reply:
That would mean one more step to actually release though. Let's try as is and we can add this additional step later when needed? Tbh, I would like to trigger the release from Jenkins anyway, just don't know how to update the versions etc.